src/python/pants/goal/run_tracker.py

```diff
@@ -615,7 +615,9 @@ def _get_options_to_record(self) -> dict:
             scopes = self._all_options.known_scope_to_info.keys()
         for scope in scopes:
             scope_and_maybe_option = scope.split("^")
-            recorded_options[scope] = self._get_option_to_record(*scope_and_maybe_option)
+            recorded_options[scope or "GLOBAL"] = self._get_option_to_record(
+                *scope_and_maybe_option
+            )
```

Comment:
How about instead using:

```
if scope == GLOBAL_SCOPE:
  scope = "GLOBAL"
```

`GLOBAL_SCOPE` is defined in `pants.option.scope`. It'd make this a bit more explicit.